### DISCUSSION TITLE
(notepadplusplus) How to force install Release Candidate

### DISCUSSION THREAD

**Discussion by superbonaci** [chocolatey-community/chocolatey-packages#1772] [Q&A]
I have installed `Notepad++ v8.1.9.3` with `choco`, however wish to install Release Candidate for 8.1.9.4, which is not available over chocolatey How can I "upgrade" manually using choco command line? This is meant to fix important issues.
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10968

**Comment by AdmiringWorm** [comment]
As this is phrased as a question, and no template is being used I will be moving this over to Discussions.

**Comment by AdmiringWorm** [comment]
A package needs to exist for the version that you wish to install, without a package version being available you aren't able to use Chocolatey to install a non-existing version.

This package version can be provided by yourself if you need it on a different Chocolatey source (be it a local source, or somewhere else) by taking the existing scripts/metadata located in this repository, updating the scripts to the version you want and adding it to that different source.

The only other way is for us to provide the package version as part of our update process, however, we do not prioritize unstable package versions so this may be unlikely at this point.

**Comment by superbonaci** [comment]
Where can I find such script?

**Reply by TheCakeIsNaOH** [reply]
https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/notepadplusplus.install

You would download the files, edit the nuspec with the new version number, download the new installers to the tools directory, edit the `chocolateyinstall.ps1` to point to the new installer filenames, pack up the package with `choco pack` and then you can install it locally or via another custom Chocolatey source.

**Reply by superbonaci** [reply]
Thanks. I've found also that just running the regular RC installer overwrites the same files installed by chocolatey, so it serves me perfectly fine and I'm on the newer version already. When v8.1.9.4 comes out I guess it will just overwrite the files and that's it.

**Reply by AdmiringWorm** [reply]
As the package on the community repository uses the official installer under the hood, it should indeed overwrite the files (assuming the official installer doesn't change to not do that anymore).

**Comment by superbonaci** [comment]
Notepad++ v8.2 already available to `chocolatey`.
